core/templates/dev/head/pages/exploration_player/PlayerConstants.js

```diff
@@ -22,6 +22,8 @@ oppia.constant('TWO_CARD_THRESHOLD_PX', 960);
 
 oppia.constant('CONTINUE_BUTTON_FOCUS_LABEL', 'continueButton');
 
+oppia.constant('EVENT_NEW_CARD_AVAILABLE','newCardAvailable');
```

Comment:
I suggest adding a one-line comment above this to explain what it represents, and doing the same for the other event, and putting these two constants next to each other (to make it clearer for future developers). Thanks!

Reply:
Ok, will do that.

Reply:
Done.